packages/wallet/src/SingleAddressWallet.ts

```diff
@@ -205,7 +205,7 @@ export class SingleAddressWallet implements Wallet {
     this.utxo = createUtxoTracker({
       addresses$,
       retryBackoffConfig,
-      store: stores.utxo,
+      store: { unspendableUtxo: stores.unspendableUtxo, utxo: stores.utxo },
```

Comment:
Let's rename this since it's multiple stores. Could as well just use the existing `stores` object instead of creating a new one then.
```suggestion
      stores,
```